src/components/header.js

```diff
@@ -5,27 +5,31 @@ import extendedLogo from '../files/icons/ca-extended-logo.svg';
 import menuIcon from '../files/icons/menu-icon.svg';
 import PropTypes from 'prop-types';
 
-const Header = ({ onMenuClick }) => {
+const Header = ({ onMenuClick, mobileMenuHiddenBool }) => {
+  console.log("mobileMenuHidden", mobileMenuHiddenBool);
```

Comment:
@alexiscott This looks great, just needs a debug statement cleanup here. Feel free to merge once that is done.

Reply:
Thanks, fixed.

Reply:
@iris-i Merged.